New Simulation

parthamishra1996 opened the pull request.

parthamishra1996 commented:
@shivamvats Is it ok now?

shivamvats commented:
@parthamishra1996  @Shaswat27 There are issues with the motion of the bot. Has there been any progress?

parthamishra1996 commented:
@shivamvats Please check now. I have removed the cmake since it isn't giving build errors. Also since we worked on simulation on separate repo, I have replaced the entire file with new changed file, in a single commit.

shivamvats commented:
@parthamishra1996 What are the 99k lines you have removed? Are they all cmake degerated?
